auth/external/goth.go

```diff
@@ -44,7 +45,7 @@ func SetupGothProviders(log *zap.Logger, redirectUrl string, ep ...settings.Prov
 			redirect = strings.Replace(redirectUrl, "{provider}", pc.Handle, 1)
 		}
 
-		if strings.HasPrefix(pc.Handle, OIDC_PROVIDER_PREFIX) {
+		if strings.HasPrefix(pc.Handle, types.OIDC_PROVIDER_PREFIX) {
```

Comment:
I don't want this to be moved and have cross component (auth <-> system) dependency.
If there is a better location for this constant, I'm all ears.

Reply:
If there is a common package then it should go there. I'm new to the code base so I've just kept it private to the package and added a comment referencing the other constant.